sal_interfaces/Scheduler/Scheduler_Events.xml

```diff
@@ -1177,4 +1177,110 @@
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>mode</EFDB_Name>
+      <Description>Current observing mode as a human-readable string.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>typeValue</EFDB_Name>
+      <Description>Id of the type of observation.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>type</EFDB_Name>
+      <Description>Type of observation as a human-readable string.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_logevent_blockStatus</EFDB_Topic>
+    <Description>Status of an individual block.</Description>
+    <item>
+      <EFDB_Name>id</EFDB_Name>
+      <Description>Id of the block (e.g. OBS-123).</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>statusId</EFDB_Name>
+      <Description>Id of the status.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>status</EFDB_Name>
+      <Description>Status of the block, as a human readable string.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>iterationsCompleted</EFDB_Name>
+      <Description>Number of completed iterations.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>iterationsTotal</EFDB_Name>
+      <Description>Total number of iterations.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>hash</EFDB_Name>
+      <Description>A unique identifier for this block.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>configuration</EFDB_Name>
+      <Description>The full set of configurations for this block.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_logevent_blockInventory</EFDB_Topic>
+    <Description>Inventory of all the available blocks.</Description>
+    <item>
+      <EFDB_Name>ids</EFDB_Name>
+      <Description>Comma-separated list of block ids.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>status</EFDB_Name>
```

Comment:
I was expecting that there would be 1 parameter per status type.
How would your format this as a single string?

OBS-123 (Ready), OBS-234 (completed) etc?  That'd be challenging to parse.

Reply:
The ids would be in the [ids](https://github.com/lsst-ts/ts_xml/blob/2a137cc26a532c572147b68118ea3682712735bd/sal_interfaces/Scheduler/Scheduler_Events.xml#L1272) attribute and the status here, e.g.;

```
ids="OBS-123,OBS-234"
status="ready,completed"
```

The problem of having keyword for the status is that we can't update them if we realized we missed something..

Reply:
fair, but if there are 20 obs IDs and you care about the status of the 10th one then it'll be tough...
Maybe we leave it like this but change it once we think we have the status' correct?

Reply:
Hopefully there will be a view in LOVE to parse this information into a nice format.. but certainly, once we have a better understanding about how we use this, we can change it up..